spdx/v2/v2_2/tagvalue/reader/parse_creation_info_test.go

```diff
@@ -78,6 +78,18 @@ func TestParserCIMovesToOtherLicenseAfterParsingLicenseIDTag(t *testing.T) {
 	}
 }
 
+func TestParserCIMovesToOtherLicenseAfterParsingDocumentTag(t *testing.T) {
```

Comment:
I think this name might be a little misleading, should it be `TestDocumentComment` or something more directly related to that?

Reply:
I agree, typo, will fix in the following PR.